modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -28,18 +19,13 @@ You can configure resource limits and requests for the following core platform m
 * Prometheus Operator and its admission webhook service
 * Telemeter Client
 * Thanos Querier
-// end::CPM[]
 
-// tag::UWM[]
 You can configure resource limits and requests for the following components that monitor user-defined projects:
 
 * Alertmanager
 * Prometheus
 * Thanos Ruler
-// end::UWM[]
 
 By defining resource limits, you limit a container's resource usage, which prevents the container from exceeding the specified maximum values for CPU and memory resources.
 
-By defining resource requests, you specify that a container can be scheduled only on a node that has enough CPU and memory resources available to match the requested resources.
-
-
+By defining resource requests, you specify that a container can be scheduled only on a node that has enough CPU and memory resources available to match the requested resources.
```

Comment:
Just a nitpick and not related to this PR:
`By defining resource requests`
could read more easily if the ambiguity were removed like so:
`By defining the resource requests`
(to make it clear that the resource requests are being defined and are not defining)
Same for `By defining resource limits` two lines above.